efficientdet/dataloader.py

```diff
@@ -333,12 +308,8 @@ def dataset_parser(self, value, example_decoder, anchor_labeler, params):
       image_scale = input_processor.image_scale_to_original
       boxes *= image_scale
       is_crowds = tf.cast(is_crowds, dtype=tf.float32)
-      boxes = pad_to_fixed_size(boxes, -1, [self._max_instances_per_image, 4])
-      is_crowds = pad_to_fixed_size(is_crowds, 0,
-                                    [self._max_instances_per_image, 1])
-      areas = pad_to_fixed_size(areas, -1, [self._max_instances_per_image, 1])
-      classes = pad_to_fixed_size(classes, -1,
-                                  [self._max_instances_per_image, 1])
+      is_crowds = tf.expand_dims(is_crowds, axis=1)
+      areas = tf.expand_dims(areas, axis=1)
```

Comment:
How faster is this new dataloader way?
I feel the code is much harder to read than before. If it is not significantly faster, maybe we can stick to the simple way for better readability?